src/main/java/com/prgms/vouchermanager/contorller/front/FrontController.java

```diff
@@ -0,0 +1,155 @@
+package com.prgms.vouchermanager.contorller.front;
+
+
+import com.prgms.vouchermanager.contorller.customer.CustomerController;
+import com.prgms.vouchermanager.contorller.customer.CustomerMenuType;
+import com.prgms.vouchermanager.contorller.voucher.VoucherController;
+import com.prgms.vouchermanager.contorller.voucher.VoucherMenuType;
+import com.prgms.vouchermanager.contorller.wallet.WalletController;
+import com.prgms.vouchermanager.contorller.wallet.WalletMenuType;
+import com.prgms.vouchermanager.util.io.ConsoleInput;
+import com.prgms.vouchermanager.util.io.ConsoleOutput;
+import com.prgms.vouchermanager.validation.InputValidation;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Controller;
+
+import static com.prgms.vouchermanager.exception.ExceptionType.*;
+import static com.prgms.vouchermanager.exception.ExceptionType.INVALID_CUSTOMER_MENU;
+
+
+@Controller
+public class FrontController {
+
+    private final static Logger logger = LoggerFactory.getLogger(FrontController.class);
```

Comment:
자바 규약에 맞는 순서대로 작성해주시면 좋을 것 같습니다.
```suggestion
    private static final Logger logger = LoggerFactory.getLogger(FrontController.class);
```

그리고 롬복의 `@Slf4j`를 쓰는 경우와 `LoggerFactory`를 둘다 사용하시는데 이유가 궁금합니다.

Reply:
처음에 LoggerFactory를 사용하다가 @Slf4j 방식으로 변경을 했는데, 
기존 LoggerFactory를 변경하지 않아 일관성이 없는 것 같습니다ㅜㅜ.
일관성있게 변경하겠습니다!